prometheus/src/cluster_metrics.rs

```diff
@@ -0,0 +1,57 @@
+use solana_gossip::cluster_info::ClusterInfo;
+use solana_runtime::bank::Bank;
+
+use crate::{
+    token::Lamports,
+    utils::{write_metric, Metric, MetricFamily},
+};
+use std::{io, sync::Arc, time::SystemTime};
+
+pub fn write_cluster_metrics<W: io::Write>(
+    at: SystemTime,
+    bank: &Arc<Bank>,
+    cluster_info: &Arc<ClusterInfo>,
+    out: &mut W,
+) -> io::Result<()> {
+    let identity_pubkey = cluster_info.id();
+    let version = cluster_info
+        .get_node_version(&identity_pubkey)
+        .unwrap_or_default();
+
+    write_metric(
+        out,
+        &MetricFamily {
+            name: "solana_cluster_identity_public_key_info",
+            help: "The current node's identity",
```

Comment:
```suggestion
            help: "The node's current identity",
```
The identity can change at runtime!